### PR TITLE
chore: remove aks handler

### PR DESCRIPTION
**Pull request by fals**
- remove aks handler
- vulnerabilities fixed with package update
- use distroless image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.17.1
+FROM static-debian11
```
Comment (mjudeikis) [inline Dockerfile:1]: Description says distroless but this is not?

Comment (fals) [reply Dockerfile:1]: @mjudeikis what am I missing here? this list says it is 🤔 https://github.com/GoogleContainerTools/distroless

Comment (fals) [reply Dockerfile:1]: We use this same image in other repos BTW

**Review by Narunas-K — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (mjudeikis) [reply Dockerfile]: You added a full path ;) problem with short path is that it defaults to docker hub and image might not be from the original publisher :)

**Pull request merged**